packages/algob/sample-project/js/test/sample-test.js

```diff
@@ -36,6 +37,10 @@ describe("Sample Test", function () {
 		assert.equal(fundReceiver.balance(), minBalance);
 		assert.equal(master.balance(), masterBalance);
 
+		// In case you want to get account from algob.config.js with network default
```

Comment:
I am not sure if its a good place to put the description at

Reply:
I think I put the wrong position. I will put them in the `this.beforeEach
`